src/webfaf/reports.py

```diff
@@ -1014,8 +1015,8 @@ def attach():
 
 def get_avg_count(first, last, count):
     diff = last - first
-    r_d = diff.days / 30.4  # avg month size
+    r_d = diff.days // 30.4  # avg month size
```

Comment:
Just /

Reply:
You are right, it should stay ```/``` because the second operator is ```float```.